task_41_hangman_recreation/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UV-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./styles.css">
+    <title>Hangman</title>
+</head>
+
+<body>
+    <h1>Hangman</h1>
+    <p>Find the hidded word - Enter a letter</p>
+    <div class="game">
+        <svg height="250" width="200" class="hangMan">
+            <!-- Rod -->
+            <line x1="60" y1="20" x2="140" y2="20" />
+            <line x1="140" y1="20" x2="140" y2="50" />
+            <line x1="60" y1="20" x2="60" y2="230" />
+            <line x1="20" y1="230" x2="100" y2="230" />
+            <!-- Head -->
+            <circle cx="140" cy="70" r="20" class="body-part" />
+            <!-- Body -->
+            <line x1="140" y1="90" x2="140" y2="150" class="body-part" />
+            <!--Arms-->
+            <line x1="140" y1="120" x2="120" y2="100" class="body-part" />
+            <line x1="140" y1="120" x2="160" y2="100" class="body-part" />
+            <!-- Legs -->
+            <line x1="140" y1="150" x2="120" y2="180" class="body-part" />
+            <line x1="140" y1="150" x2="160" y2="180" class="body-part" />
+
```

Comment:
be consistent on spacing. there is no need of a line break here.

Reply:
Ok. like break removed.